### ISSUE TITLE
DCGAN Batch Norm initialization

### ISSUE THREAD

**Issue by samiede** [pytorch/examples#516]
Hey everyone!

I've been playing around with the DCGAN a little, and reading the paper it says:
"All weights were initialized from a zero-centered Normal distribution with standard deviation 0.02", but in the implementation you find that the batch norm layers have in fact been initialized around 1.0. I've tried both and actually had better result with an initialization around 1, but I couldn't find any rationale for that in the literature and online.
Does anyone have an explanation for that?

Cheers!

**Comment by soumith** [comment]
no real rationale, just empirical success.
